src/Identity.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         2.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+declare(strict_types=1);
+
+/**
+ * An identity decorator implementing both Authorization\IdentityInterface and
+ * Authentication\IdentityInterface.
+ */
+namespace Authorization;
+
+use Authentication\IdentityInterface as AuthenIdentityInterface;
+
+class Identity extends IdentityDecorator implements AuthenIdentityInterface
```

Comment:
Should the docs be updated to note that this class will used by default if the other plugin is installed?

Reply:
Will do.

Reply:
Doc updated.